### PR TITLE
Build: Remove dates from copyright notice

### PR DESCRIPTION
**Pull request by agcolom**
I'm not too sure about the change in Gruntfile.js, so please let me know if you would like it to be different, e,g, whether the jquery.org/license should be on the same line as the Copyright notice, whether it should be https://jquery.org/ or https://jquery.org/license also, or anything else.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ module.exports = function( grunt ) {
 						"ascii_only": true
 					},
 					banner: "/*! jQuery v<%= pkg.version %> | " +
-						"(c) 2005, <%= grunt.template.today('yyyy') %> jQuery Foundation, Inc. | " +
+						"Copyright jQuery Foundation and other contributors " +
```
Comment (mzgol) [inline Gruntfile.js]: Why the change from `(c)` to `Copyright`? You didn't do the same in the Sizzle PR: https://github.com/jquery/sizzle/pull/307/files

This is the banner put in the minified files, e.g. https://code.jquery.com/jquery-2.1.3.min.js

Comment (mzgol) [inline Gruntfile.js]: I'd leave `|` at the end, it separates the copyright info from the URL to the license (see the linked minified file for example).

Comment (agcolom) [inline Gruntfile.js]: @mzgol Thanks for the comments. I'll make the changes now.

**Comment by mzgol** [comment]
LGTM

File: Gruntfile.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ module.exports = function( grunt ) {
 						"ascii_only": true
 					},
 					banner: "/*! jQuery v<%= pkg.version %> | " +
-						"(c) 2005, <%= grunt.template.today('yyyy') %> jQuery Foundation, Inc. | " +
+						"(c) jQuery Foundation, Inc. | " +
```
Comment (staabm) [inline Gruntfile.js]: In the intro file sou also dropped the `Inc`

Comment (markelog) [inline Gruntfile.js]: +1, why this is being dropped btw?

Comment (dmethvin) [inline Gruntfile.js]: We don't need them, according to lawyers. So removing the dates makes it easier to maintain! :smiley:

Comment (dmethvin) [inline Gruntfile.js]: I'll remove the `Inc.` here for consistency.
